### PR TITLE
rework the way we use the expiry links

### PR DESCRIPTION
**Pull request by zadigus**
[Recap of our workflow](https://github.com/softozor/shopozor-backend#pull-requests)

# Prerequisites

- [x] You have configured the triggering of the pre-commit hook on your local repository's clone. You need to have installed the python module `pre-commit` (listed in [the required dev modules](requirements-dev.txt)) and run the command `pre-commit install` in the repository's root folder.
- [x] You have covered your code with unit and acceptance tests
- [x] The feature you have implemented has no tag `@wip` any more

# Changes

Out of the old `ExpiryLink.json` fixture, I created the `PasswordResetExpiryLink.json` and `SignupExpiryLink.json` fixtures. This is because the frontend applications need those fixtures like that for the sake of testing. I kept the backend acceptance test code as is, I've just reworked the fixture injection, so that the common checks can still be performed without code duplication.

### CODE REVIEW

**Comment by zadigus** [comment]
ok to test

**Comment by zadigus** [comment]
ok to test

File: features/fixtures.py [hunk 1]
```diff
@@ -165,15 +165,15 @@ def signup_password_not_compliant(context):
     data = json.load(
         os.path.join(settings.GRAPHQL_RESPONSES_FOLDER, 'Authentication', 'SignupPasswordNotCompliant.json'))
     password_not_compliant = data['data']['consumerCreate']['errors']
-    context.password_not_compliant = password_not_compliant
+    context.password_not_compliant = password_not_compliant[0]
```
Comment (kili236) [inline features/fixtures.py]: [Suggestion, consistency] To be consistent with line 176, I would replace theses lines by:
```password_not_compliant = data['data']['consumerCreate']['errors'][0]
context.password_not_compliant = password_not_compliant```

Comment (kili236) [inline features/fixtures.py]: [Suggestion, consistency] To be consistent with line 176, I would change these two lines by:
`password_not_compliant = data['data']['consumerCreate']['errors'][0]`
and 
`context.password_not_compliant = password_not_compliant`

Comment (zadigus) [reply features/fixtures.py]: Good point.

**Pull request merged**